packages/admin-ui-docs/docusaurus.config.js

```diff
@@ -30,6 +30,10 @@ const config = {
         theme: {
           customCss: require.resolve('./src/css/custom.scss'),
         },
+        gtag: {
+          trackingID: 'G-K6LT0G8PH7',
```

Comment:
Isn't this sensitive content?

Reply:
This is just the id to where we are sending our tracking data and this id will also appears in the request when the data is sent to GA.

Reply:
It's a "public" ID available in all source codes on html pages, no problem.